### PR TITLE
feat: New progress bar to replace old bootstrap one

### PR DESCRIPTION
**Pull request by arpargo**
# Replacing old Bootstrap ProgressBar

### KER-299

----------------------------------------------------------------------------------------------

Reason behind change:
Removing old, deprecated React-bootstrap library from use and switching to HDS components. HDS did not offer suitable component for replacing the ProgressBar so instead created a custom component to solve this need.

### CODE REVIEW

File: src/components/QuestionResults.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { v1 as uuid } from 'uuid';
-import { ProgressBar } from 'react-bootstrap';
+import ProgressBar from './ProgressBar';
```
Comment (mikkojamG) [inline src/components/QuestionResults.jsx:5]: Lint errors here, import order 👀

File: src/components/admin/QuestionForm.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { v1 as uuid } from 'uuid';
-import { ProgressBar } from 'react-bootstrap';
+import ProgressBar from '../ProgressBar';
```
Comment (mikkojamG) [inline src/components/admin/QuestionForm.jsx:6]: Lint errors here, import order 👀

File: assets/sass/kerrokantasi/_common.scss [hunk 3]
```diff
@@ -251,4 +251,18 @@ a.list-group-item {
...
+  height: 1.4em;
+  position: relative;
+  width: 100%;
+  border-radius: 5px;
+  background: $light-gray;
+  .progressBar {
+    height: 100%;
+    position: relative;
+    border-radius: inherit;
+    background: blue;
```
Comment (mikkojamG) [inline assets/sass/kerrokantasi/_common.scss:266]: I think should use --color-coat-of-arms to keep original color 🤔

**Review by mikkojamG — CHANGES_REQUESTED** [review]
Couple of comments, nothing major 🙂

File: assets/sass/kerrokantasi/_common.scss (hunk 3 above)
Comment (arpargo) [reply assets/sass/kerrokantasi/_common.scss:266]: Actually the original color comes straight from bootstrap but you're right, I should add Helsinki theme styles and use coat-of-arms there 👍

**Pull request merged**